src/dune_rules/cram/cram_rules.ml

```diff
@@ -4,7 +4,8 @@ open Memo.O
 module Spec = struct
   type t =
     { loc : Loc.t
-    ; alias : Alias.Name.Set.t
+    ; alias : Alias.Name.t option
```

Comment:
How about a name like `test_name_alias`? Also, why is it an option?

Reply:
Renamed and option removed.